source/dicom/valuerep.py

```diff
@@ -47,7 +47,17 @@ class DSfloat(float):
 
     """
     __slots__ = 'original_string'
-
+    def __getstate__(self):
```

Comment:
I actually don't think this is doing what you think it's doing anyways. `__slots__` is a string so you don't want to iterator over it like you are.